application/services/Reports/getTbAllSitesResultsSheet.sql

```diff
@@ -0,0 +1,860 @@
+SELECT
+    flattenedevaluationresults.`country`,
+    flattenedevaluationresults.`site NO.`,
+    flattenedevaluationresults.`site name / location`,
+    flattenedevaluationresults.`pt - id`,
+    flattenedevaluationresults.`submitted`,
+    flattenedevaluationresults.`submission excluded`,
+    flattenedevaluationresults.`DATE pt received`,
+    flattenedevaluationresults.`DATE pt results reported`,
+    JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .cartridge_lot_no") AS `cartridge lot number`,
+    flattenedevaluationresults.`assay_name` AS `assay`,
+    CASE
+        WHEN
+            JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .expiry_date") = '0000-00-00'
+        THEN
+            NULL
+        ELSE
+            COALESCE(
+                STR_TO_DATE(JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .expiry_date"), '%d-%b-%Y'),
+                STR_TO_DATE(JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .expiry_date"), '%Y-%b-%d'),
+                STR_TO_DATE(JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .expiry_date"), '%d-%m-%Y'),
+                STR_TO_DATE(JSON_UNQUOTE(flattenedevaluationresults.attributes_json -> " $ .expiry_date"), '%Y-%m-%d')
+            )
+    END AS `expiry DATE`,
+    flattenedevaluationresults.`DATE OF last instrument calibration`,
```

Comment:
The casing has been changed on a lot of these column names, assuming this is as a result of some pattern based find & replace work but as an example, this column should be called `Date of last instrument calibration`, not `DATE OF last instrument calibration`. Please try and revert the column names to what they were?

Reply:
Yeah that's the work of a SQL formatter, will restore column names